examples/evolve_dstream_surf.py

```diff
@@ -29,16 +29,28 @@ def make_uuid4_fast() -> str:
     return str(uuid.UUID(int=random.getrandbits(128), version=4))
 
 
-def evolve_drift(population: typing.List) -> typing.List:
+def evolve_drift(
+    population: typing.List,
+    fossil_interval: typing.Optional[int] = None,
+    fossil_sample_percentage: float = 0.1
+) -> typing.List:
     """Simple asexual evolutionary algorithm under drift conditions."""
```

Comment:
Given the audience for `examples/` (new users) we should probably briefly explain what a `fossil` is in this context the `fossil_interval` and `fossil_sample_percentage`

(this example is already a bit underexplained/undercommented of my own doing, so feel free to add other notes that would be useful in a first-time read)